src/state/store.ts

```diff
@@ -0,0 +1,20 @@
+import { configureStore } from '@reduxjs/toolkit'
+import playPausePlaylistReducer from './slices/playPausePlaylistSlice'
+import playlistTempoReducer from './slices/playlistTempoSlice'
+import playlistMetronomeEnabledReducer from './slices/playlistMetronomeEnabledSlice'
+import playlistRuntimeReducer from './slices/playlistRuntimeSlice'
+
+export const store = configureStore({
+  reducer: {
+    playlistPlaying: playPausePlaylistReducer,
+    playlistTempo: playlistTempoReducer,
+    playlistMetronomeEnabled: playlistMetronomeEnabledReducer,
+    playlistRuntime: playlistRuntimeReducer
```

Comment:
These all seem like the same class of state - can we merge these into one compositional slice? This level of granularity will become hard to maintain at scale

Reply:
Yea that’s a fire idea, wasn’t sure how to structure it

Reply:
this was actually the best idea ever... just got it done